### PR TITLE
Add in support for arrays of observables

### PR DESCRIPTION
**Pull request by misaizdaleka**
Hi, 

I made the required changes and added a test to support it. However, I didn't have a chance to run the test, so please make sure it's working properly. 

Let me know if I should change anything. 

Thanks

### CODE REVIEW

File: src/auto-unsubscribe.ts [hunk 1]
```diff
@@ -2,20 +2,23 @@ function isFunction( fn ) {
   return typeof fn === 'function';
 }
 
-export function AutoUnsubscribe( {blackList = []} = {}) {
+export function AutoUnsubscribe2({ blackList = [], includeArrays = false } = {}) {
```
Comment (NetanelBasal) [inline src/auto-unsubscribe.ts]: You changed the name to `AutoUnsubscribe2`

**Comment by NetanelBasal** [comment]
Looks good. Just fix the name, and if you update the README, it will be great.

File: src/auto-unsubscribe.ts (hunk 1 above)
Comment (misaizdaleka) [reply src/auto-unsubscribe.ts]: I did fix that in subsequent commit, I think you should see it.

**Comment by NetanelBasal** [comment]
Thanks! Published 2.1.0.

**Pull request merged**